### PR TITLE
Blog post/proposal: unified Haskell installer

### PR DESCRIPTION
**Pull request by snoyberg**
Rendered version at https://deploy-preview-20--snoyman-com.netlify.app/blog/2021/05/unified-haskell-installer/

### CODE REVIEW

File: content/blog/unified-haskell-installer.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+Let me clarify that I'm wearing two hats with what I'm about to say:
+
+1. A maintainer of Stack and the original author of much of Stack's toolchain install logic, who would always rather see the Stack codebase size go down instead of up.
+2. A member of the Haskell Foundation board, who is looking to improve the Haskell adoption story.
+
+So the problem statement: can't we standardize an install method that meets most people's goal, most of the time?
+
+## The goal
+
+I'll start by laying out the ultimate goal for in my head. This idea's been bouncing around for a few months now, but only came up in May 4, 2021 tech track meeting for the Haskell Foundation. I don't think it's that ambitious of a goal. But in the next section, I'm also going to lay out some shorter term milestones.
```
Comment (myShoggoth) [inline content/blog/unified-haskell-installer.md]: 'goal for in my head' is a little awkward.

Comment (snoyberg) [reply content/blog/unified-haskell-installer.md]: Worse than awkward, it's just wrong. Good catch, thanks!

**Comment by myShoggoth** [comment]
Love it! Do you want to be more explicit about seeking volunteers to take on pieces of the work? Discussing and collaborating might be too soft.

**Comment by snoyberg** [comment]
I was trying to avoid the word "volunteers," because it sounds like "come work on someone else's project." I'm _really_ looking for people to come and take _ownership_ of this, which you're rightfully pointing out isn't captured in what I've said. I think perhaps saying it that bluntly would be the correct thing to do.

**Comment by cdornan** [comment]
This is awesome. So awsome, and clearly needed, that I think we should just crack on while trusting and making it clear that we are looking for bright new things to get involved.

**Pull request merged**